www/layouts/shortcodes/layout-list.html

```diff
@@ -19,6 +19,7 @@
     "colemak-french-touch" "Colemak French Touch"
     "bepolar"     "Bépolar"
     "bepo"        "Bépo"
+    "hypergol"    "Hypergol"
```

Comment:
Same here : unless I’m missing a point, I think this should come up with another PR.